packages/proxy/test/unit/http/response-middleware.spec.ts

```diff
@@ -213,7 +234,7 @@ describe('http/response-middleware', function () {
           ...props.res,
         },
         req: {
-          proxiedUrl: 'http:127.0.0.1:3501/multi-domain.html',
+          proxiedUrl: 'http://127.0.0.1:3501/multi-domain.html',
```

Comment:
Turns out this wouldn't be parsable by the `cors.urlMatchesOriginPolicyProps` helper. It doesn't have an effect in this test since the remote strategy is set to `foo`, but figured it would be good to update this incase these tests are updated in the future and this helps discover a potential issue